Sources/Plot/API/HTMLComponents.swift

```diff
@@ -579,6 +579,72 @@ extension List: ComponentContainer where Items == ComponentGroup {
     }
 }
 
+// Component used to render a `<picture>` element for responsive image support.
+public struct Picture: Component {
+
```

Comment:
Code style nitpick: Please remove this blank line to make this type definition consistent with the rest of the library.

Reply:
Will do.